app/views/candidate_interface/start_page/eligibility.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title, title_with_error_prefix(t('page_titles.eligibility'), @eligibility_form.errors.any?) %>
-<% content_for :before_content, govuk_back_link_to(candidate_interface_start_path) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_create_account_or_sign_in_path) %>
```

Comment:
No longer relevant as this page is behind a feature flag, but looks like this back link was wrong!